docs/site/Decorators.md

```diff
@@ -132,56 +132,181 @@ accordingly or do a composition of them:
+  - name: name of the parameter, should be a `string`, 
 
+  An equivalent example using shortcut decorator would be:
+  
   ```ts
-    // example 2: decorator `@param` applied on parameter level
     class MyController {
-      @get('/')
+      @get('/Pets/{category}')
       list(
-        @param(offsetSpec) offset?: number,
-        @param(pageSizeSpec) pageSize?: number,
+        @param.path.string('category') category: string,
+        @param.query.number('pageSizes') pageSize?: number,
       ) {}
     }
   ```
 
-  In the first example, we apply multiple decorators to a single declaration.
-  The order in which the decorators are called is important because a `@param`
-  decorator must be applied after an operation decorator.
-  To learn more about TypeScript decorator composition,
-  refer to [TypeScript Decorator Documentation](https://www.typescriptlang.org/docs/handbook/decorators.html)
+  You can find the specific usage in [Writing Controller methods](controller.htm#writing-controller-methods)
+
+  *We are supporting parameter location 'cookie' in the feature, track the feature in story*
+  *https://github.com/strongloop/loopback-next/issues/997*
 
-  Please note method level `@param` and parameter level `@param` are mutually exclusive,
-  you can not mix and apply them to the same parameter.
+### RequestBody Decorator
 
-  You can also use this pattern to make it quicker to define params: `@param.${in}.${type}(${name})`
+  Syntax: see [API document](https://github.com/strongloop/loopback-next/blob/master/packages/openapi-v3/src/request-body-decorator.ts#L20-L79)
 
-  - in: one of the following values: `query`, `header`, `path`, `formData`, `body`
-  - type: one of the following values:  `string`, `number`, `boolean`, `integer`
-  - name: a `string`, name of the parameter
+  `@requestBody()` is applied to a method parameter to generate OpenAPI requestBody specification for it.
+  
+  *Only one parameter can be decorated by `@requestBody` per controller method.*
+
+  A typical OpenAPI requestBody spec contains property `description`, `required`, and `content`:
+
+  ```ts
+    requestBodySpec: {
+    description: 'a user',
+    required: true,
+    content: {
+      'application/json': {...schemaSpec},
+      'application/text': {...schemaSpec},
+    },
+  }
+  ```
+
+  When the type of decorated argument can be described as a schema, you can define 
+  it as a model class with `@model` and `@property`:
+  
+  ```ts
+  import {model, property} from '@loopback/repository';
+  import {Address} from './address';
+
+  @model()
+  class User {
+    @property()
+    firstname: string
+    @property()
+    lastname: string
+    @property()
+    address: Address
+  }
+  ```
+  *About decorating models and the corresponding OpenAPI schema, please check*
+  *[model decorators](#model-decorators)*
+
+  Then a typical usage of `@requestBody` is specifying a parameter's type as the model class,
```

Comment:
`A typical usage of '@requestBody' is specifying a parameter as a requestBody by decorating it` or something along that line.

Reply:
@shimks  I still prefer to keep "then" :)

There is some code snippet in between, but the context is:
> When the decorated argument is a custom type or a model, we recommend you define 
  a model class with `@model` and `@property`. Then a typical usage of `@requestBody` is specifying a parameter's type as the model class, and applying the decorator on it.